css/src/components/selectionCard.css

```diff
@@ -23,8 +23,11 @@
 }
 
 .Selection-card--disabled {
-  pointer-events: none;
+  border-radius: var(--spacing-m);
+  position: relative;
+  cursor: not-allowed;
   box-shadow: inset 0 0 0 var(--spacing-xs) var(--secondary-lighter);
+  transition: var(--duration--fast-01) var(--standard-productive-curve);
```

Comment:
Instead of repeating the css properties in both `Selection-card` and `Selection-card--disabled` please move all the common properties into one css class